### PR TITLE
Added log rage for one-liner request formatting

### PR DESCRIPTION
**Pull request by msmirnoff**

### CODE REVIEW

**Comment by alvinypyim** [comment]
You are moving the code from the projects 👍

File: lib/production_toolkit/initializers/lograge.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+if defined?(Rails)
+  if Rails.env.production?
+    require 'lograge'
+    Rails.application.class.configure do
```
Comment (alxberardi) [inline lib/production_toolkit/initializers/lograge.rb:4]: Why does it need `application.class` ?

**Comment by mnoack** [comment]
:+1:

**Comment by alxberardi** [comment]
:+1:

**Pull request merged**